product_state/views/product_template.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="product_state_search_form_view">
+        <field name="name">product.state.search.form</field>
+        <field name="model">product.state</field>
+        <field name="arch" type="xml">
+            <search string="Product State">
+                <field name="name" />
+            </search>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="action_open_state_products">
+        <field name="name">State Products</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">product.template</field>
+        <field name="view_mode">kanban,form,tree</field>
+        <field name="context">{}</field>
+        <field name="domain">[('product_state_id', '=', active_id)]</field>
+    </record>
+    <record model="ir.actions.act_window" id="action_open_single_product_state">
+        <field name="name">Product State</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">product.state</field>
+        <field name="view_mode">kanban,form,tree</field>
+        <field name="context">{}</field>
+        <field name="domain">[('id', '=', product_state_id)]</field>
+        <field name="target">current</field>
+    </record>
+    <record id="view_product_state_form" model="ir.ui.view">
+        <field name="name">product.state.form</field>
+        <field name="model">product.state</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="%(action_open_state_products)d"
+                            type="action"
+                            class="oe_stat_button"
+                            icon="fa-cubes"
+                        >
+                            <field
+                                name="products_count"
+                                widget="statinfo"
+                                string="Products"
+                            />
+                        </button>
+                    </div>
+                    <div class="oe_title">
+                        <label for="name" string="State Name" class="oe_edit_only" />
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                    </div>
+                    <group string="Code">
+                        <field name="code" nolabel="1" />
+                    </group>
+                    <group string="Description">
+                        <field name="description" nolabel="1" />
+                        <field name="default" widget="boolean_toggle" />
+                        <field name="active" widget="boolean_toggle" />
```

Comment:
The alignment in this group seems wrong:
![image](https://user-images.githubusercontent.com/1033124/204814300-52527a29-067f-4a1b-b002-cba830688e54.png)

Can you fix this?

Reply:
Fixed